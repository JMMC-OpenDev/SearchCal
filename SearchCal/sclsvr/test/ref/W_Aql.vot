version https://git-lfs.github.com/spec/v1
oid sha256:0a1b9c77eaad163574cf2f70d20983a4f99d1b9726dbb93fc26da7bc89ab412c
size 126466
