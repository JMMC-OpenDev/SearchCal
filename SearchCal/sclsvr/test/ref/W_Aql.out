version https://git-lfs.github.com/spec/v1
oid sha256:421222ab799c2462370873531960c3530a5decb38145c92166af8a4039629ccd
size 160078
